apps/users/models.py

```diff
@@ -1,20 +1,21 @@
-from core.models import BaseModel
 from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
 from django.db import models
 
-from .queryset import UserManager
+from apps.core.models import BaseModel
+from apps.users.queryset import UserManager
 
 
 class User(BaseModel, AbstractBaseUser, PermissionsMixin):
-    first_name = models.CharField(max_length=1000, null=True, blank=True)
-    last_name = models.CharField(max_length=1000, null=True, blank=True)
-    username = models.CharField(max_length=1000, unique=True)
-    email = models.EmailField(unique=True)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255, null=True, blank=True)
+    phone_number = models.CharField(max_length=50)
+    age = models.IntegerField()
```

Comment:
`age` should be `PositiveIntegerField()`

Reply:
what difference between ? anyway frontend or backend can check this field, if needed.